src/main/webapp/js/messages-loader.js

```diff
@@ -4,17 +4,17 @@ function fetchMessages(){
   const url = '/feed';
   fetch(url).then((response) => {
     return response.json();
-}).then((messages) => {
+  }).then((messages) => {
     const messageContainer = document.getElementById('message-container');
-  if(messages.length == 0){
-    messageContainer.innerHTML = '<p>There are no posts yet.</p>';
-  }
-  else{
-    messageContainer.innerHTML = '';
-  }
-  messages.forEach((message) => {
-    const messageDiv = buildMessageDiv(message);
-  messageContainer.appendChild(messageDiv);
+    if(messages.length == 0){
+      messageContainer.innerHTML = '<p>There are no posts yet.</p>';
+    }
+    else{
+      messageContainer.innerHTML = '';
+    }
+    messages.forEach((message) => {
+      const messageDiv = buildMessageDiv(message);
+    messageContainer.appendChild(messageDiv);
```

Comment:
```suggestion
    messageContainer.appendChild(buildMessageDiv(message));
```
local var not necessary.